### ISSUE TITLE
Write tests for utils in v0.1

### ISSUE THREAD

**Issue by Sh0g0-1758** [BlocSoc-iitr/Athena#10]
Write a test checking the implementation of starkent_keccak. 

You can use this test : 

```
        >>> from nethermind.starknet_abi.utils import starknet_keccak
        >>> starknet_keccak(b"transfer").hex()
        '0083afd3f4caedc6eebf44246fe54e38c95e3179a5ec9ea81740eca5b482d12e'

```

**Comment by Rishikpulhani** [comment]
pls assign

**Comment by Sh0g0-1758** [comment]
Closed via #12
